Classes/Steno.sc

```diff
@@ -243,15 +243,14 @@ Steno {
 		var n;
 		if(busIndices.isNil) {
 			// allocate busses for maxBracketDepth plus fadeBus (used in filter fades)
-			n = numChannels * (maxBracketDepth + 1);
+			n = numChannels * maxBracketDepth;
 			busIndices = server.audioBusAllocator.alloc(n);
-			if(busIndices.isNil) {
+			fadeBus = server.audioBusAllocator.alloc(numChannels);
+			if(busIndices.isNil || {fadeBus.isNil}) {
```

Comment:
I usually leave blank spaces inside the `{`. Maybe you can do the same? Not a big thing of course.